src/js/pre-need/config/form.js

```diff
@@ -461,10 +461,25 @@ const formConfig = {
                   )
                 }
               },
-              hasCurrentlyBuried: {
+              veteranHasBuried: {
                 'ui:title': 'Is there anyone currently buried in a VA National Cemetery under your eligibility?',
+                'ui:required': formData => isVeteran(formData),
                 'ui:widget': 'radio',
                 'ui:options': {
+                  hideIf: (formData) => !isVeteran(formData),
+                  labels: {
+                    1: 'Yes',
+                    2: 'No',
+                    3: 'I don\'t know',
+                  }
+                }
+              },
+              sponsorHasBuried: {
```

Comment:
You could make these one field by using `updateSchema` to switch between the different titles and enum options.

Reply:
Great idea @jbalboni, thanks for pointing that out. This has been updated.